core/src/bundle/usBundlePrivate.cpp

```diff
@@ -121,10 +121,10 @@ BundlePrivate::BundlePrivate(const std::shared_ptr<Bundle>& qq, CoreBundleContex
     bundleManifest.SetValue(Bundle::PROP_AUTOLOAD_DIR, Any(this->info.autoLoadDir));
   }
 
-#ifdef US_ENABLE_AUTOLOADING_SUPPORT
+#if defined(US_ENABLE_AUTOLOADING_SUPPORT) && defined(US_BUILD_SHARED_LIBS)
```

Comment:
re-visit whether we need ```defined(US_BUILD_SHARED_LIBS)```. This would mean that if CppMicroServices was statically linked in, auto-installing based on bundle meta-data wouldn't work.